pkg/timeutil/timeutil.go

```diff
@@ -26,3 +42,22 @@ func LastWeekday(weekday time.Weekday) time.Time {
 
 	return date
 }
+
+// StringToWeekday maps a string to a time.Weekday.
+func StringToWeekday(day string) (time.Weekday, error) {
+	timeWeekday, ok := weekday[strings.ToLower(day)]
+	if !ok {
+		return time.Sunday, fmt.Errorf("timeutil: Invalid weekday '%s' given", day)
+	}
+
+	return timeWeekday, nil
+}
+
+// FormatDuration prints the time using the given format.
+func FormatDuration(duration time.Duration, timeFormat string) string {
+	if timeFormat == "decimal" {
```

Comment:
I agree mostly, but I'm not sure about putting it in `types`, I think it'd be fine in this file really. I was thinking of `types` as a package for "API types" so to speak. Imagine if this were exposed a REST API, they would be the types used.

This is a great suggestion though for avoiding magic strings and promoting code re-use.